must-gather.Dockerfile

```diff
@@ -2,8 +2,7 @@ FROM registry.redhat.io/openshift4/ose-must-gather-rhel9:v4.18@sha256:b998280f89
 
 FROM registry.redhat.io/ubi9-minimal@sha256:ac61c96b93894b9169221e87718733354dd3765dd4a62b275893c7ff0d876869
 
-#RUN microdnf install tar rsync
-
+RUN microdnf install -y tar rsync
```

Comment:
do you know where `rsync` is being used? I can't see it used in the commands. Perhaps with `oc adm inspect`?

Reply:
it's the oc adm must-gather command itself that will use rsync and tar inside the container to fetch out the files (without these two inside it is not able to get the files, I tried)